HW2/Task5.py

```diff
@@ -0,0 +1,15 @@
+'''
+You have a group of people with non-unique names. Generate a list of non-duplicate names (you cannot use set for this
+task. If there are 2 johns in the list, you need to take only one of them.
+"John Dow", "John Dow", "Marta Dow" => "John Dow", "Marta Dow ")
+'''
+
+group_of_people = ['John', 'Erica', 'Sam', 'John', 'John', 'John', 'Ben', 'Ben']
+
+unique_names_list = []
+
+for name in group_of_people:
+    if group_of_people.count(name) == 1:
+        unique_names_list.append(name)
```

Comment:
as I mentioned at the lesson you should use only default functions for slowing tasks.